pkgs/applications/misc/pastel/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, rustPlatform, Security }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pastel";
+  version = "v0.5.3";
```

Comment:
Can the version only have the numeric value and then the `v` can be added below

Reply:
Fixed.